reference/matrix/csr_kernels.cpp

```diff
@@ -783,14 +783,9 @@ void convert_to_hybrid(std::shared_ptr<const ReferenceExecutor> exec,
          i++) {
         for (size_type j = 0; j < result->get_ell_stride(); j++) {
             result->ell_val_at(j, i) = zero<ValueType>();
-            result->ell_col_at(j, i) = 0;
+            result->ell_col_at(j, i) = invalid_index<IndexType>();
         }
     }
-    for (size_type i = 0; i < result->get_coo_num_stored_elements(); i++) {
```

Comment:
coo may have more space than the input

Reply:
why is that?

Reply:
when converting to hybrid, it always recompute the config not preserve any higher allowance. Thus, it's not a issue currently.

Reply:
I see! IMO, Coo should never have padding entries, so this didn't really occur to me.